components/LineChart.js

```diff
@@ -0,0 +1,346 @@
+              text: tick => ({ x: -9, y: 5, text: tick }),
+              gridline: { x2: innerWidth },
+            })}
+          </g>
+
+          <g
+            className="rightAxis"
+            textAnchor="start"
+            transform={`translate(${innerWidth}, 0)`}
+            key="right"
+          >
+            {plotTicks({
+              scale: chartData.yScaleLine,
+              tickNum: 3,
+              x: () => 0,
+              y: (scale, tick) => scale(tick),
+              roundFn: value => Math.ceil(value),
+              text: tick => ({ x: 8, y: 5, text: tick }),
+              gridline: {},
+            })}
+          </g>
+
+          <g
+            className="bottomAxis"
+            textAnchor="middle"
+            transform={`translate(0, ${innerHeight})`}
+            key="bottom"
+          >
+            {plotTicks({
+              scale: chartData.xScale,
+              tickNum: 11,
+              x: (scale, tick) => scale(tick),
+              y: () => 0,
+              roundFn: d => startOfDay(d),
+              text: tick => ({ x: 0, y: 20, text: formatDate(tick) }),
+              gridline: { y2: -innerHeight },
+            })}
+          </g>
+
+          <g className="webGroup">
+            <path className="line webLine" d={chartData.webLine(dataset)} />
+          </g>
+          <g className="chatbotGroup">
+            <path
+              className="line chatbotLine"
+              d={chartData.chatbotLine(dataset)}
+            />
+          </g>
+        </g>
+      </svg>
+      <div
+        className={classes.wrapper}
+        style={{ left: margin.left + 'px', top: margin.top + 'px' }}
+      >
+        {dataset.map((d, i) => (
+          <div
+            className="hitbox"
+            key={i}
+            style={{
+              left: boxWidth * (i - 0.5) + 'px',
+              width: boxWidth + 'px',
+              height: innerHeight + 'px',
+            }}
+          >
+            <div
+              className="vLine"
+              style={{ height: innerHeight + 'px', left: boxWidth / 2 + 'px' }}
+            ></div>
+            <div
+              className="dot web"
+              style={{
+                top: chartData.yScaleWeb(d.webVisit) - 5 + 'px',
+                left: boxWidth / 2 - 5 + 'px',
+              }}
+            ></div>
+            <div
+              className="dot chatbot"
+              style={{
+                top: chartData.yScaleLine(d.lineVisit) - 5 + 'px',
+                left: boxWidth / 2 - 5 + 'px',
+              }}
+            ></div>
+            <div className={`tooltip ${i < 15 ? 'left' : 'right'}`}>
+              <div className="tooltip-title">{formatDate(d.date, true)}</div>
+              <div className="tooltip-body">
+                <p className="tooltip-subtitle">{t`Web Visit`}</p>
+                <p className="tooltip-text">
+                  <span>{d.webVisit}</span>{' '}
+                  <span>{c('LineChart').t`times`}</span>
```

Comment:
We can use [ngettext](https://ttag.js.org/docs/ngettext.html) to handle plural form.
```
c('LineChart').ngettext(msgid`${num} time`, `${num} times`, num)
```